freqtrade/optimize/optimize_reports.py

```diff
@@ -225,7 +225,7 @@ def generate_daily_stats(results: DataFrame) -> Dict[str, Any]:
 
 def generate_backtest_stats(btdata: Dict[str, DataFrame],
                             all_results: Dict[str, Dict[str, Union[DataFrame, Dict]]],
-                            min_date: Arrow, max_date: Arrow
+                            min_date: Arrow, max_date: Arrow, processed: Dict
```

Comment:
This will not make sense.

The (non-analyzed) backtest data is available as btdata should the raw candles be necessary - and keeping this data around will cause memory leaks when used with multiple strategies (`--strategy-list`).

Reply:
Agree :+1:  I don't know way I didn't saw this. Maybe because, I was on an older code-base while implementing it. I will remove it.